sale_procurement_group_by_line/model/sale.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2013-2014 Camptocamp SA - Guewen Baconnier
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.tools.float_utils import float_compare
+from odoo.exceptions import UserError
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    @api.model
+    def _prepare_procurement_group_by_line(self, line):
+        """ Hook to be able to use line data on procurement group """
+        return {'name': line.order_id.name}
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    @api.multi
+    def _get_procurement_group_key(self):
+        """ Return a key with priority to be used to regroup lines in multiple
+        procurement groups
+
+        """
+        return 8, self.order_id.id
+
+    @api.multi
+    def _action_launch_procurement_rule(self):
+        """
+        Create procurements based on quantity ordered.
+        If the quantity is increased, new procurements are created.
+        If the quantity is decreased, no automated action is taken.
+        """
+        precision = self.env['decimal.precision'].\
```

Comment:
add
```python
res = super(SaleOrderLine, self)._action_launch_procurement_rule()
```
and then end the method with a `return res` instead of `return True`